tutorials/31_Metadata_Filtering.ipynb

```diff
@@ -0,0 +1,966 @@
+{
```

Comment:
<p>Do all document stores support metadata filtering? We can add this as a warning here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Good question, I'll check

Reply:
They all implement it, pgvector in development